export/distro/azure.go

```diff
@@ -0,0 +1,84 @@
+//
```

Comment:
This file is almost the same as mqtt.go. Why duplicate the file instead of using only one?

Reply:
I agree. Since there is no suitable Azure SDK for devices written in the Go, sending MQTT will work. Someday when proper SDK is released, we can update this feature.